modules/custom_operations/examples/calculate_grid/export_model.py

```diff
@@ -0,0 +1,42 @@
+# Copyright (C) 2018-2022 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import numpy as np
+import argparse
+import torch
+import torch.nn as nn
+from torch.autograd import Variable
+from .calculate_grid import CalculateGrid
+
+
+class MyModel(nn.Module):
+    def __init__(self):
+        super(MyModel, self).__init__()
+        self.calculate_grid = CalculateGrid()
+
+    def forward(self, x):
+        return self.calculate_grid.apply(x)
+
+
+def export(num_points, max_grid_extent):
+    # Generate a list of unique positions and add a mantissa
+    np.random.seed(32)
+    torch.manual_seed(11)
+
+    inp_pos = np.random.randint(0, max_grid_extent, [num_points, 3])
+    inp_pos = torch.tensor(inp_pos) + torch.rand(inp_pos.shape, dtype=torch.float32) # [0, 1)
+
+    model = MyModel()
+    with torch.no_grad():
+        torch.onnx.export(model, (inp_pos), 'model.onnx',
```

Comment:
It should be as simple as `ov_model = convert_model(sparse_conv, example_input=(features, inp_pos, out_pos, voxel_size), extensions='libuser_cpu_extension.so')`. With the current state of OV master it doesn't work, because we are not passing `ONNX_FALLTHROUGH` by default inside `convert_model`, and for some reason I need to pass the full path to extension lib (even if it is placed in the current directory -- @ilyachur, @popovaan, is it expected?). Anyway with the already mentioned in another comment single line modification in `convert_model` this piece of code works with the current master and provides OpenVINO model with all extensions applied.

Note, that with `conver_model` we are not assigning names for inputs/outputs providing the capability to access them as positional arguments/results. Names of inputs/outputs look to be pure ONNX-export artifact here.

Reply:
Still it is not heavily tested, that's why we are not providing this extensibility story for users.

Reply:
In general we don't provide a limitation for extensions loading, and it should be possible to load extension with using relative path.

Reply:
@ilyachur, what do you mean? It's not a part of extensions, this script just generates the onnx model. It's needed only for tests and to show how to export torch model with an unsupported operation.